src/pydata_sphinx_theme/locale/es/LC_MESSAGES/sphinx.po

```diff
@@ -7,133 +7,131 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PROJECT VERSION\n"
 "Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
-"POT-Creation-Date: 2023-02-16 14:32-0500\n"
-"PO-Revision-Date: 2023-02-16 13:23-0500\n"
+"POT-Creation-Date: 2023-04-14 11:36+0200\n"
+"PO-Revision-Date: 2023-04-14 10:04+0200\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language: es\n"
 "Language-Team: es <LL@li.org>\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Generated-By: Babel 2.11.0\n"
+"Generated-By: Babel 2.12.1\n"
 
 #: src/pydata_sphinx_theme/theme/pydata_sphinx_theme/layout.html:50
 msgid "Skip to main content"
-msgstr "Saltar al contenido principal"
+msgstr ""
```

Comment:
why delete all the translations in this file?

Reply:
because I played with the parameters of Babel. To see the diff I needed to nuke the files. I'll reintegrate the translations in a downstream commit